src/main/java/io/swagger/template/TemplateApi.java

```diff
@@ -13,114 +13,197 @@
-
-                    Parameter[] parameters = method.getParameters();
-                    int j=0;
-                    if (parameters.length != 0) {
-                        for(Parameter parameter:parameters){
-                            if(j>0){
-                                stringBuilderParameter.append(",");
-                            }
-                            parameterList=parameterList.replaceAll("parameterdata",((RequestParam)parameter.getAnnotations()[2]).value());
-                            parameterList=parameterList.replaceAll("parameter",method.getParameterTypes()[0].getName()+" "+((RequestParam) method.getParameters()[0].getAnnotations()[2]).value());
-                            stringBuilderParameter.append(parameterList);
-                            j++;
-                            }
-                        integrate = integrate.replaceAll("datatype", parameterList);
-                        integrate = integrate.replaceAll("integrationspecific", "return new ResponseEntity<String>(" + ((String) iterator.get(integrationName)).toLowerCase() + "Proxy." + method.getName() +"("+((RequestParam) method.getParameters()[0].getAnnotations()[2]).value()+"), HttpStatus.OK);");
-
-                        proxy=proxy.replaceAll("path","\""+((RequestMapping)classNameInterfaces[0].getMethods()[i].getAnnotations()[2]).value()[0]+"\"");
-                        i++;
-                        proxy=proxy.replaceAll("parameter","@RequestParam(\""+((RequestParam)method.getParameters()[0].getAnnotations()[2]).value()+"\")"+method.getParameterTypes()[0].getName()+" "+((RequestParam) method.getParameters()[0].getAnnotations()[2]).value());
-                    } else {
-                        integrate = integrate.replaceAll("datatype", "");
-                        integrate = integrate.replaceAll("integrationspecific", "return new ResponseEntity<String>(" + ((String) iterator.get(integrationName)).toLowerCase() + "Proxy." + method.getName() + "(), HttpStatus.OK);");
-
-                        proxy=proxy.replaceAll("path","\""+((RequestMapping)classNameInterfaces[0].getMethods()[i].getAnnotations()[2]).value()[0]+"\"");
-                        i++;
-                        proxy=proxy.replaceAll("parameter","");
-                    }
-                    stringBuilderIntegrate.append(integrate);
-                    stringBuilderIntegrate.append("\n");
-
-
-                    stringBuilderProxy.append(proxy);
-                    stringBuilderProxy.append("\n");
-                } else {
-                    break;
-                }
+  public static void main(String[] args) throws Exception {
+
+    IntegrationName integrationName = new IntegrationName();
+    Field[] fieldList = integrationName.getClass().getFields();
+
+    for (Field iterator : fieldList) {
+
+      //Script to generate  CONTROLLER CODE(IntegrationNameAPI.java).
+      String content = IOUtils.toString(new FileReader(new File("TemplateName.java")));
```

Comment:
i wonder if you should call the file something other than .java - specifically something that let's people know it's not really a Java file that's meant to be used as is.  maybe something more like ".java.template" so it's clear this is a template of a java file.

Reply:
I have removed some **redundant** files and **renamed** some the  **template** files appropriately in the recent commit on prateek.